installer/compose/README.md

```diff
@@ -90,6 +91,18 @@ docker-compose -f docker-compose.full.yml down
 #docker-compose -f docker-compose.full.yml down
 ```
 
+We introduce quickstart deployment that comes with a set of predefined StreamPipes assets, to use the quickstart mode, just build the Docker image and start **quickstart** option:
+```bash
+docker-compose -f docker-compose.quickstart.yml build script-runner
+docker-compose -f docker-compose.quickstart.yml up -d
+# go to after all services are started http://localhost
```

Comment:
```suggestion
# go to `http://localhost` after all services are started 
```

Reply:
Hi, @bossenti !
I have changed the descriptiong of this row, as well as the same description for 'Nats' and 'Full' option.